<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fontly</class>
 <widget class="QWidget" name="fontly">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>fontly</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/* FONT *
/* app name */
QLabel#fontlyLabel{
	font: 75 18pt &quot;Roboto&quot;;
}
/* text inputs */
QLineEdit#fontApply, QLineEdit#search {
	font: 57 14pt &quot;Roboto&quot;;
}
/* sections */
QLabel#menuLabel , QLabel#socialLabel {
	font: 57 14pt &quot;Roboto Medium&quot;;
}
/* subsections */
QLabel#fontPickerLabel, QLabel#myFontsLabel, QLabel#settingsLabel, QFrame#githubLabel, QLabel#facebookLabel, QLabel#websiteLabel  {
	font: 75 13pt &quot;Roboto Medium&quot;;
}


/* ICONS */
QPushButton#close {
	image: url(:/assets/icons/closeINNACTIVE.png);
}
QPushButton#maximize {
	image: url(:/assets/icons/maximizeINNACTIVE.png);
}
QPushButton#minimize {
	image: url(:/assets/icons/minimizeINNACTIVE.png);
}
QPushButton#textColor {
	image: url(:/assets/icons/textColor.png);
}
QPushButton#textBackgroundColor {
	image: url(:/assets/icons/textBackgroundColor.png);
}
QPushButton#resetSettings {
	image: url(:/assets/icons/resetSettings.png);
}
QFrame#fontPickerIcon {
	image: url(:/assets/icons/fontpicker.png);
}
QFrame#myFontsIcon {
	image: url(:/assets/icons/myfonts.png);
}
QFrame#settingsIcon {
	image: url(:/assets/icons/settings.png);
}
QFrame#socialGithubIcon {
	image: url(:/assets/icons/socialGITHUB.png);
}
QFrame#socialFacebookIcon {
	image: url(:/assets/icons/socialFB.png);
}
QFrame#socialWebsiteIcon {
	image: url(:/assets/icons/socialWWW.png);
}
QFrame#welcomeWindowBackground {
	image: url(:/assets/icons/welcomeWindow.png);
}
QFrame#searchIconBackground {
	image: url(:/assets/icons/search.png);
}
/* BACKGROUND  COLOR */

/* transparency */
QFrame, QPushButton{
	background-color: rgba(0,0,0,0);
}
/* main color: darker */
QFrame#actionButtonsArea {
	background-color: rgb(38, 50, 56);
}
/* main color: lighter */
QFrame#toolsArea, QFrame#bottomBorder {
	background-color: rgb(55, 71, 79);
}
/* white */
QTextEdit, QLineEdit, QListWidget#textScroll, QFrame#appBackgroundPanel{
	background-color: rgb(255,255,255);
}
/* white but darkerr */
QFrame#leftPanelArea {
	background-color: rgb(245, 245, 245);
}
/* accent elements */
QFrame#searchIconBackground, QFrame#appNameBackground{
	background-color: #e74c3c;
}

/* TEXT COLOR */

/* white */
QLabel#fontlyLabel {
  color:  #fff;
}
/* white but darker */
QLabel#fontPickerLabel {
	color: rgb(245,245,245);
}
/* black 50% opacity */
QLabel#menuLabel , QLabel#socialLabel {
  color: rgba(0,0,0,.5);
}
/* black 35% opacity */
QLabel#myFontsLabel, QLabel#settingsLabel, QFrame#githubLabel, QLabel#facebookLabel, QLabel#websiteLabel , QLabel#fontPickerLabel{
  color: rgba(0,0,0,.35);
}
/* BORDER COLOR */

/* transparent */
QListView, QPushButton {
	border: 1px solid rgba(0,0,0,0);
}
/* BORDER RADIUS */

/* both sides */
QLineEdit#fontApply {
    border-radius: 20%;
}
/* right side */
QFrame#searchIconBackground {
	border-top-right-radius: 20%;
	border-bottom-right-radius: 20%;
}
QFrame#appNameBackground {
	border-top-right-radius: 25%;
	border-bottom-right-radius: 25%;
}
/* left side */
QLineEdit#search {
	border-color:rgba(0,0,0,0);
	border-top-left-radius: 20%;
	border-bottom-left-radius: 20%;
}

/* SLIDER */
QSlider::groove:horizontal {
	border-radius: 4px;
    height: 8px;
    background-color: #263238;
}

QSlider::handle:horizontal {
    background-color: #e74c3c;
    border: 1px solid #e74c3c;
    width: 15px;
	margin: -4px 0;
    border-radius: 4px;
}

/* ON HOVER */
QPushButton#fontPickerButton:hover, QPushButton#myFontsButton:hover, QPushButton#settingsButton:hover, QPushButton#githubButton:hover, QPushButton#facebookButton:hover, QPushButton#websiteButton:hover{
	border-left: 8px solid rgb(38, 50, 56);
}
QPushButton#fontPickerButton:pressed, QPushButton#myFontsButton:pressed, QPushButton#settingsButton:pressed,QPushButton#githubButton:pressed, QPushButton#facebookButton:pressed, QPushButton#websiteButton:pressed {
  border-left: 2px solid rgb(38, 50, 56);
}
QPushButton#close:hover {
	image: url(:/assets/icons/close.png);
}

QPushButton#maximize:hover {
	image: url(:/assets/icons/maximize.png);
}

QPushButton#minimize:hover {
	image: url(:/assets/icons/minimize.png);
}

QPushButton#close:pressed {
	image: url(:/assets/icons/closeINNACTIVE.png);
}

QPushButton#maximize:pressed {
	image: url(:/assets/icons/maximizeINNACTIVE.png);
}

QPushButton#minimize:pressed {
	image: url(:/assets/icons/minimizeINNACTIVE.png);
}</string>
  </property>
  <widget class="QFrame" name="leftPanelArea">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>140</y>
     <width>260</width>
     <height>530</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="menuLabel">
    <property name="geometry">
     <rect>
      <x>35</x>
      <y>50</y>
      <width>221</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Menu</string>
    </property>
   </widget>
   <widget class="QFrame" name="fontPickerIcon">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>103</y>
      <width>25</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="fontPickerLabel">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>105</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Font Picker</string>
    </property>
   </widget>
   <widget class="QLabel" name="myFontsLabel">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>155</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>My Fonts</string>
    </property>
   </widget>
   <widget class="QFrame" name="myFontsIcon">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>153</y>
      <width>25</width>
      <height>25</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QLabel" name="settingsLabel">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>194</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Settings</string>
    </property>
   </widget>
   <widget class="QFrame" name="settingsIcon">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>203</y>
      <width>25</width>
      <height>25</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QLabel" name="socialLabel">
    <property name="geometry">
     <rect>
      <x>35</x>
      <y>285</y>
      <width>221</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Check this project on</string>
    </property>
   </widget>
   <widget class="QLabel" name="githubLabel">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>340</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>GitHub</string>
    </property>
   </widget>
   <widget class="QLabel" name="facebookLabel">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>392</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Facebook</string>
    </property>
   </widget>
   <widget class="QFrame" name="socialWebsiteIcon">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>440</y>
      <width>25</width>
      <height>25</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QFrame" name="socialGithubIcon">
    <property name="geometry">
     <rect>
      <x>45</x>
      <y>334</y>
      <width>41</width>
      <height>30</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QLabel" name="websiteLabel">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>442</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Website</string>
    </property>
   </widget>
   <widget class="QFrame" name="socialFacebookIcon">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>390</y>
      <width>25</width>
      <height>25</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="fontPickerButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>90</y>
      <width>260</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="myFontsButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>140</y>
      <width>260</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="settingsButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>190</y>
      <width>260</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="facebookButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>380</y>
      <width>260</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="githubButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>330</y>
      <width>260</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="websiteButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>430</y>
      <width>260</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QFrame" name="fontPickerIsActive">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>90</y>
      <width>260</width>
      <height>50</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QFrame" name="myFontsIsActive">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>140</y>
      <width>260</width>
      <height>50</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QFrame" name="settingsIsActive">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>190</y>
      <width>260</width>
      <height>50</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <zorder>myFontsIsActive</zorder>
   <zorder>settingsIsActive</zorder>
   <zorder>fontPickerIsActive</zorder>
   <zorder>menuLabel</zorder>
   <zorder>fontPickerIcon</zorder>
   <zorder>fontPickerLabel</zorder>
   <zorder>myFontsLabel</zorder>
   <zorder>myFontsIcon</zorder>
   <zorder>settingsLabel</zorder>
   <zorder>settingsIcon</zorder>
   <zorder>socialLabel</zorder>
   <zorder>githubLabel</zorder>
   <zorder>facebookLabel</zorder>
   <zorder>socialWebsiteIcon</zorder>
   <zorder>socialGithubIcon</zorder>
   <zorder>websiteLabel</zorder>
   <zorder>socialFacebookIcon</zorder>
   <zorder>fontPickerButton</zorder>
   <zorder>myFontsButton</zorder>
   <zorder>settingsButton</zorder>
   <zorder>facebookButton</zorder>
   <zorder>githubButton</zorder>
   <zorder>websiteButton</zorder>
  </widget>
  <widget class="QFrame" name="bottomBorder">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>670</y>
     <width>1281</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(55, 71, 79);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QFrame" name="actionButtonsArea">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>40</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="close">
    <property name="geometry">
     <rect>
      <x>1240</x>
      <y>10</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="maximize">
    <property name="geometry">
     <rect>
      <x>1210</x>
      <y>10</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="minimize">
    <property name="geometry">
     <rect>
      <x>1180</x>
      <y>10</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="toolsArea">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>40</y>
     <width>261</width>
     <height>100</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QFrame" name="appNameBackground">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>25</y>
      <width>130</width>
      <height>50</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="fontlyLabel">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>4</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Fontly&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QFrame" name="appBackgroundPanel">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>720</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>640</width>
     <height>320</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1280</width>
     <height>720</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QStackedWidget" name="currentWindow">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>40</y>
      <width>1020</width>
      <height>630</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="welcomeWindow"/>
   </widget>
  </widget>
  <zorder>appBackgroundPanel</zorder>
  <zorder>leftPanelArea</zorder>
  <zorder>bottomBorder</zorder>
  <zorder>actionButtonsArea</zorder>
  <zorder>toolsArea</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
